### PR TITLE
SONARPY-2462: Parallelize the qa_plugin tests

### PR DESCRIPTION
**Pull request by Seppli11**
[SONARPY-2462](https://sonarsource.atlassian.net/browse/SONARPY-2462)

[SONARPY-2462]: https://sonarsource.atlassian.net/browse/SONARPY-2462?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java [hunk 1]
```diff
@@ -17,38 +17,34 @@
...
-import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 import org.sonarqube.ws.Issues;
 
 import static com.sonar.python.it.plugin.TestsUtils.issues;
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Disabled
```
Comment (guillaume-dequenne-sonarsource) [inline its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java:30]: Was that test not run before? And it's actually failing?

File: its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java [hunk 2]
```diff
@@ -17,38 +17,34 @@
...
   private static final String PROJECT_KEY = "ipynb_json_project";
 
   @RegisterExtension
-  public static final OrchestratorExtension ORCHESTRATOR = TestsUtils.ORCHESTRATOR;
+  public static final ConcurrentOrchestratorExtension ORCHESTRATOR = TestsUtils.ORCHESTRATOR;
 
-  @BeforeAll
-  static void startServer() {
+  @Test
+  void test() {
```
Comment (guillaume-dequenne-sonarsource) [inline its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java:39]: Running the test locally, the issues do look legit. We could update the assertions (I believe asserting an empty list of issue was a mistake).
We can do that here or in a separate PR, as you prefer.

Comment (Seppli11) [reply its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java]: I'll change the assertion. I think it should be doable in this PR

File: its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java (hunk 1 above)
Comment (Seppli11) [reply its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/NotebookPluginTest.java]: As far as I can tell, it wasn't run before. I've tried running this test on master and it failed as well. So, it seems that it failing is not connected to changes made in this PR.

File: its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/ConcurrentOrchestratorExtension.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+  public SonarScanner createSonarScanner() {
+    return SonarScanner.create();
+  }
+
+  public static ConcurrentOrchestratorExtensionBuilder builderEnv() {
+    return new ConcurrentOrchestratorExtensionBuilder(Configuration.createEnv());
+  }
+
+  public static ConcurrentOrchestratorExtensionBuilder builder(Configuration config) {
```
Comment (guillaume-dequenne-sonarsource) [inline its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/ConcurrentOrchestratorExtension.java:64]: This seems unused, do we need it?

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
Small comment on unsued method.
LGTM otherwise, great work!

File: its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/ConcurrentOrchestratorExtension.java (hunk 3 above)
Comment (Seppli11) [reply its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/ConcurrentOrchestratorExtension.java:64]: Good catch. You're right, we don't need it. thx
